Cargo.toml

```diff
@@ -71,6 +71,7 @@ integration-ebpf = { path = "test/integration-ebpf", default-features = false }
 lazy_static = { version = "1", default-features = false }
 libc = { version = "0.2.105", default-features = false }
 log = { version = "0.4", default-features = false }
+netns-rs = "0.1"
```

Comment:
could you do the usual { version = .., default-features = false } please?

Reply:
Sure. Done in 8ec9f73

Reply:
Sure. Done in 8ec9f73